build.gradle

```diff
@@ -120,7 +120,7 @@ sonarqube {
     property "sonar.projectName", "Reform :: reform-scan-notification-service"
     property "sonar.projectKey", "uk.gov.hmcts.reform:reform-scan-notification-service"
     property "sonar.coverage.jacoco.xmlReportPaths", "${jacocoTestReport.reports.xml.destination.path}"
-    property "sonar.exclusions", "**/config/**"
+    property "sonar.exclusions", "**/config/**,**/model/**"
```

Comment:
just a note: can be removed once tests are present

Reply:
yeep